platform.md

```diff
@@ -61,6 +61,8 @@ The build image MUST ensure that:
 
 - The image config's `User` field is set to a non-root user with a writable home directory.
 - The image config's `Env` field has the environment variable `PACK_STACK_ID` set to the stack ID.
+- The image config's `Env` field has the environment variable `PACK_USER_ID` set to the uid of the `User`.
+- The image config's `Env` field has the environment variable `PACK_GROUP_ID` set to the primary gid of the `User`.
```

Comment:
- `gid` -> `GID`
- "of the `User`" -> "associated with the user specified in the `User` field"